descriptions/api.github.com/api.github.com.yaml

```diff
@@ -48194,6 +48235,166 @@ components:
       required:
       - url
       - status
+    pages-health-check:
+      title: Pages Health Check Status
+      description: Pages Health Check Status
+      type: object
+      properties:
```

Comment:
@xuorig @imwiss can you please make sure the response properties that are guaranteed to be present are required in the JSON Schema?

Reply:
Thanks @gr2m, we're going to fix this and add a linter for this 👍